internal/controller/nonadmin_controller.go

```diff
@@ -147,9 +149,21 @@ func ensureRequiredLabels(deploymentObject *appsv1.Deployment) {
 }
 
 func ensureRequiredSpecs(deploymentObject *appsv1.Deployment, dpa *oadpv1alpha1.DataProtectionApplication, image string, imagePullPolicy corev1.PullPolicy) error {
-	namespaceEnvVar := corev1.EnvVar{
-		Name:  "WATCH_NAMESPACE",
-		Value: deploymentObject.Namespace,
+	envVars := []corev1.EnvVar{
+		{
+			Name:  "WATCH_NAMESPACE", // TODO: fix, this is only used to indicate oadp ns, and is not actually used for watching ns.
+			Value: deploymentObject.Namespace,
+		},
+	}
+	if dpa.Spec.Configuration != nil && dpa.Spec.Configuration.Velero != nil {
```

Comment:
Perhaps I should have an else that if not specified then default to info

Reply:
lets have same default as velero, since we using same option for both

Reply:
Current default level is from the:
   https://github.com/migtools/oadp-non-admin/commit/5204eb4fe7a0b72d3eacdfe4db6f2258e80391f1#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R85
   
   Is this answering your question?

Reply:
yes